### PR TITLE
Add github actions steps, and pre-fix any rubocop issues

### PR DESCRIPTION
**Pull request by jarednielson**
With github actions running the tests and rubocop are really easy

Seems worth it...

Also seems like our linters should actually pass rubocop? IDK

### CODE REVIEW

File: .github/workflows/rubocop.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+name: Rspec
```
Comment (brentsondgeroth) [inline .github/workflows/rubocop.yml:1]: 🍝

File: .github/workflows/rubocop.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+    env:
+      BUNDLE_WITHOUT: "default doc job cable storage ujs test db"
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Ruby 2.7
+        uses: ruby/setup-ruby@v1
+        with:
+          bundler-cache: true
+      - name: Run Rspec
```
Comment (brentsondgeroth) [inline .github/workflows/rubocop.yml:25]: 🍝

File: .github/workflows/rubocop.yml (hunk 1 above)
Comment (jarednielson) [inline .github/workflows/rubocop.yml:1]:
```suggestion
name: Rubocop
```

File: .github/workflows/rubocop.yml (hunk 2 above)
Comment (jarednielson) [inline .github/workflows/rubocop.yml:25]:
```suggestion
      - name: Run Rubocop
```

File: .github/workflows/rspec.yml [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    branches:
+      - master
+  pull_request:
+    branches:
+      - '**'
+    paths:
+      - '**.rb'
+
+jobs:
+  rubylint:
```
Comment (brentsondgeroth) [inline .github/workflows/rspec.yml:14]: rspec?

File: .github/workflows/rubocop.yml [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+    branches:
+      - master
+  pull_request:
+    branches:
+      - '**'
+    paths:
+      - '**.rb'
+
+jobs:
+  rubylint:
```
Comment (brentsondgeroth) [inline .github/workflows/rubocop.yml:14]: Should we match the name of the workflow?

File: .github/workflows/rspec.yml (hunk 3 above)
Comment (jarednielson) [inline .github/workflows/rspec.yml:14]:
```suggestion
  rspec:
```

File: .github/workflows/rubocop.yml (hunk 4 above)
Comment (jarednielson) [inline .github/workflows/rubocop.yml:14]:
```suggestion
  rubocop:
```

**Review by brentsondgeroth — APPROVED** [review]

**Pull request merged**